tests/drivers/build_all/video/testcase.yaml

```diff
@@ -22,3 +22,6 @@ tests:
   drivers.video.mcux_smartdma.build:
     platform_allow:
       - frdm_mcxn947/mcxn947/cpu0
+  drivers.video.stm32_dcmi.build:
+    platform_allow: stm32h7b3i_dk
+    extra_args: SHIELD=st_b_cams_omv_mb1683
```

Comment:
There it is I think:
https://github.com/zephyrproject-rtos/zephyr/actions/runs/14917714935/job/41907115938#step:12:348